docs/user/library/data/shape.rst

```diff
@@ -37,7 +37,7 @@ The following connection parameters are available:
 | "timezone"              | Optional: Timezone used to parse dates in the      |
 |                         | DBF file                                           |
 +-------------------------+----------------------------------------------------+
-| "memory mapped buffer"  | Optional: memory map the files (unadvisable for    |
+| "memory mapped buffer"  | Optional: memory map the files (not advised for    |
```

Comment:
I think either "inadvisable" or "not recommended" would be better here - "not advised" is not a very common phrasing.